op-scim-bridge/chart/op-scim-bridge-mp/templates/deployer-cleanup.yaml

```diff
@@ -7,16 +41,20 @@ metadata:
     app.kubernetes.io/name: {{ .Values.name }}
     app.kubernetes.io/component: cleanup
 spec:
-  schedule: "*/1 * * * *" # once a minute
-  successfulJobsHistoryLimit: 0
+  schedule: "*/10 * * * *" # every 10 minutes
+  successfulJobsHistoryLimit: 1
   failedJobsHistoryLimit: 1
   jobTemplate:
     spec:
+      parallelism: 1 # how many pods will be instantiated at once
+      completions: 1 # how many containers of the job are instantiated one after the other (sequentially) inside the pod
+      backoffLimit: 1 # maximum pod restarts in case of failure
+      activeDeadlineSeconds: 300 # limit the time for which a Job can continue to run; 5 minutes
       template:
         metadata:
           labels: *CronLabels
         spec:
-          serviceAccountName: {{ .Values.scim.serviceAccount }}
+          serviceAccountName: "{{ .Values.name }}-cleanup-sa"
```

Comment:
Where does this service account come from? Is it automatically created as needed? Or do we need to statically create one during the setup process?

Reply:
Good question. Previously we created a SA during the setup of the marketplace app. I.e. you needed to select an existing one or create a new one. The problem with this is we need specific access for the SA to be able to look up jobs (spawned by the CronJob) and delete completed ones.

I opted for the deployer cleanup to rather create what it needs during deployment and let all of this live under the same namespace, so that it does not rely on any of the other service accounts used in the deployment. This makes the cleanup process self sufficient, and less error prone.

Reply:
Thanks! I see the definition now: https://github.com/1Password/op-scim-gcp-marketplace/pull/53/files#diff-4b8928cecbd7e63893343840a6fe84b79856adcce7e1b7b7ae9c518375fcb842R1